src/App.tsx

```diff
@@ -1,26 +1,123 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useRef, useState } from 'react';
+import classNames from 'classnames';
 
-const USER_ID = 0;
+import { UserWarning } from './UserWarning';
+import { USER_ID, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './Components/TodoList';
+import { Footer } from './Components/Footer';
+import { Header } from './Components/Header';
+import { Loader } from './Components/Loader';
+import { Filter } from './types/Filter';
+import { chooseActіveArray } from './utils/functions';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filteredButton, setFilteredButton] = useState<Filter>('all');
+  const [isLoading, setIsLoading] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [makingChanges, setMakingChanges] = useState(false);
+  const [clearCompleted, setClearCompleted] = useState(false);
```

Comment:
Do you really need to keep it in the state? Why don't you run deleting by clicking on the button?